packages/devtools_shared/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Package of shared structures between devtools_app and devtools_serv
 
 # Note: this version should only be updated by running tools/update_version.sh
 # that updates all versions of packages from packages/devtools.
-version: 0.8.1-dev.1
```

Comment:
you shouldn't have to update this version as part of the CL. The publish script will now update all versions in lock step.

Reply:
reverted.